### DISCUSSION TITLE
Plugin 'code' languages

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#604] [Ideas]
It would be nice if we could restrict the languages, or types of languages, which are displayed by the code plugin, so the snippet is representative of 'real' code.  IMO it would be good if the 'random' selection was weighted by the languages most used, so it is more likely to show code in the languages the user most often uses.

**Comment by Nixinova** [comment]
You can restrict the languages using the categories `prose`,`markup`,`programming`and/or`data`

**Reply by jayvdb** [reply]
I dont see that config ability in https://github.com/lowlighter/metrics/tree/master/source/plugins/code

**Reply by Nixinova** [reply]
Oh, the code snippit plugin. No, there's not.

**Comment by lowlighter** [comment]
This could be done by using linguist-js on fetched events snippets

**Reply by lowlighter** [reply]
This has been implemented with #652
